lesson_03/quiz/src/quizzes/quizzes.module.ts

```diff
@@ -23,7 +23,7 @@ const QUIZ_PROVIDERS = [
   DasiaEnglishQuiz,
   ChigazoGrahamsQuiz,
   AmiyahJonesQuiz,
-  XavierCruzQuiz
```

Comment:
Should not delete Xavier's quiz. May be a bad merge. Just add this back and the import statement above manually and you should be good.